backend/module/eCampCore/src/Repository/CampCollaborationRepository.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace eCamp\Core\Repository;
+
+use Doctrine\ORM\EntityRepository;
+use Doctrine\ORM\NonUniqueResultException;
+use eCamp\Core\Entity\CampCollaboration;
+
+class CampCollaborationRepository extends EntityRepository {
+    public function findByInviteKey(string $inviteKey): ?CampCollaboration {
+        $q = $this->createQueryBuilder('c');
+        $q->where('c.inviteKey = :inviteKey');
+        $q->setParameter('inviteKey', $inviteKey);
+
+        try {
+            $q->setMaxResults(1);
+
+            return $q->getQuery()->getOneOrNullResult();
+        } catch (NonUniqueResultException $e) {
+            // This shouldn't ever happen
+            return null;
+        }
+    }
+
+    public function save(CampCollaboration $campCollaboration) {
+        $this->getEntityManager()->persist($campCollaboration);
+        $this->getEntityManager()->flush();
+    }
```

Comment:
This is the part that allows the RPC service to circumvent ACL, right? Is there any way we can mark this class or method so it's not used accidentally in other classes in the future? Maybe rename this method to `saveWithoutACL`?

Reply:
done